fxa-auth-db-server/lib/error.js

```diff
@@ -50,6 +50,17 @@ AppError.incorrectPassword = function () {
   )
 }
 
+AppError.sessionAlreadyRegistered = function () {
+  return new AppError(
+    {
+      code: 400,
+      error: 'Bad request',
+      errno: 104,
+      message: 'Session token is registered with a different device'
```

Comment:
Something about this naming seems a bit backward to me; in my head it's the *device* that's being registered, not the session token.  Maybe something more generic like "deviceIdConflict"?  Not sure...